src/components/announcer/CyVerseAnnouncer.js

```diff
@@ -34,6 +40,15 @@ class CyVerseAnnouncer extends Component {
     dequeue = () => {
         if (msgQueue.length > 0) {
             this.setState({ msg: msgQueue.shift(), open: true });
+            if (msgQueue.length === 0) {
+                clearInterval(this.state.timer);
+                let timer = setInterval(this.tickCallback, EMPTY_QUEUE_TIMEOUT);
+                this.setState({ timer });
+            } else {
+                clearInterval(this.state.timer);
+                let timer = setInterval(this.tickCallback, TIMEOUT);
+                this.setState({ timer });
+            }
```

Comment:
Right now we see some delay in displaying the message. This change will display the message right away When the empty queue is empty.